helpers/collectorsList.js

```diff
@@ -37,3 +37,14 @@ module.exports = {
     getCollectorIds,
     createCollector
 };
+
+/**
+ * @typedef CollectorData
```

Comment:
nicely done, I wasn't sure how to solve it. Since it has to be manually updated, could you update docs? https://github.com/duckduckgo/tracker-radar-collector#creating-new-collectors